koch.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+
+# Script to bootstrap `koch` if necessary
+#
+# Copyright (c) 2021 Leorize <leorize+oss@disroot.org>
+#
+# Licensed under the terms of the MIT license. See "license.txt" included in
+# this distribution for more information.
+
+from pathlib import Path
+from typing import Any, ClassVar, Final, final, Optional
+import os
+import shutil
+import subprocess
+import sys
+
+# Nim source directory, which is the directory this script resides in
+NimSource: Final = Path(__file__).resolve(strict=True).parent
+
+# The nimcache folder
+Nimcache: Final = NimSource / "nimcache"
```

Comment:
Not a deal beaker; I think people will want to configure this. I don't think we need to do anything yet.

Reply:
I think the line is whatever that we need before `koch.nim` can be run.